tftp/backend.py

```diff
@@ -231,6 +231,72 @@ def cancel(self):
             self.state = 'cancelled'
 
 
+@interface.implementer(IReader)
+class DynamicReader(object):
+    """A reader to go with L{FilesystemSynchronousBackend}.
+
+    @see: L{IReader}
+
+    @param file_path: relative file path, specified as part of a TFTP read request (RRQ)
+    @type file_path: C{bytes}
+
+    @param dyn_handler: a callback function to handle dynamic requests
```

Comment:
The documentation must include info w.r.t. what is expected of this callback function (what arguments it must accept, what does it have to do etc).